src/components/PathwayCourse/styles.js

```diff
@@ -71,6 +71,12 @@ const useStyles = makeStyles((theme) => ({
     alignItems: "baseline",
     paddingTop: "16px",
   },
+  certificateLogo: {
+    fontSize: "90px",
+  },
+  link: {
+    color: "green",
```

Comment:
Use theme colors for consistency in styles and to allow for future changes to be made in one place.